packages/spaces/src/lib/SpacesImage.tsx

```diff
@@ -12,6 +12,7 @@ type BaseSpacesImageProps = {
     | 'images.promotionalHover';
   fallback?: string;
   id?: string;
+  Loader?: (props: any) => JSX.Element;
```

Comment:
Since we're passing an `id` to the `Loader` I think we should set the type for props as `{ id: string }` instead of `any`

Reply:
Good call, this is fixed.